recipes/coatran/meta.yaml

```
@@ -1,43 +1,42 @@
-{% set name = "CoaTran" %}
-{% set version = "0.0.1" %}
-{% set commit = "e0d34b0" %}
-{% set sha256 = "a20c263675ec34ecbfd08eecfaa639aa5b6fb7648b5d27fdf41cec9f8eb8cce0" %}
+{% set version = "0.0.4" %}
 
 package:
-  name: "{{ name|lower }}"
-  version: "{{ version }}"
-
-source:
-  url: "https://github.com/niemasd/CoaTran/archive/{{ commit }}.zip"
-  sha256: {{ sha256}}
+  name: coatran
+  version: {{ version }}
 
 build:
-  number: 3
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('coatran', max_pin="x.x.x") }}
+
+source:
+  url: https://github.com/niemasd/CoaTran/archive/refs/tags/{{ version }}.tar.gz
+  sha256: b0af9eff5c78a7126ce5f36386c3c514e811d3470392a7a18566195e40765456
 
 requirements:
   build:
-    - make
+    - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-  host:
-  run:
-
-test:
-  commands:
-    - coatran_constant --help 2>&1 | grep CoaTran
-    - coatran_expgrowth --help 2>&1 | grep CoaTran
-    - coatran_transtree --help 2>&1 | grep CoaTran
-    - coatran_inftime --help 2>&1 | grep CoaTran
+    - make
 
 about:
-  home: "https://github.com/niemasd/CoaTran"
-  license: GPLv3
-  license_family: GPL
+  home: https://github.com/niemasd/CoaTran
+  license: GPL-3.0-or-later
+  license_family: GPL3
   license_file: LICENSE
-  summary: "CoaTran: Coalescent tree simulation along a transmission network"
-  dev_url: "https://github.com/niemasd/CoaTran"
+  summary: Coalescent tree simulation along a transmission network
+  dev_url: https://github.com/niemasd/CoaTran
+
+test:
+  commands:
+    - coatran_constant -h 2>&1 | grep "CoaTran"
+    - coatran_inftime -h 2>&1 | grep "CoaTran"
+    - coatran_transtree -h 2>&1 | grep "CoaTran"
 
 extra:
   identifiers:
     - biotools:coatran
+    - doi:10.1101/2020.11.10.377499
   recipe-maintainers:
-    - acaprez
+    - acapraz
```

```suggestion
    - acaprez
```
I think you meant `acaprez`?